streams/src/main/java/org/apache/kafka/streams/internals/ApiUtils.java

```diff
@@ -18,43 +18,58 @@
 
 import java.time.Duration;
 import java.time.Instant;
-import java.util.Objects;
+
+import static java.lang.String.format;
 
 public final class ApiUtils {
+
+    public static final String MILLISECOND_VALIDATION_FAIL_MSG_FRMT = "The \"%s\" for \"%s\" is incorrect, value: %s.";
```

Comment:
Nit: maybe `Invalid value for parameter \"%s\" (value was: %s). ` (note, I would add a space at the end (cf. comment below)